docker-compose.yml

```diff
@@ -173,7 +175,7 @@ services:
       GF_SERVER_ROOT_URL: 'https://%(domain)s/grafana/'
       GF_SMTP_ENABLED: ${TTN_DASHBOARD_GRAFANA_SMTP_ENABLED:-false}
       GF_SMTP_SKIP_VERIFY: ${TTN_DASHBOARD_GRAFANA_SMTP_SKIP_VERIFY:-false}
-      GF_SMTP_HOST: ${TTN_DASHBOARD_GRAFANA_SMTP_HOST:-localhost}:25
+      GF_SMTP_HOST: ${TTN_DASHBOARD_GRAFANA_SMTP_HOST:-localhost.com}:25
       GF_SMTP_FROM_ADDRESS: "grafana@localhost.com"
```

Comment:
I think this should be `grafana@example.com`.

Reply:
Makes perfect sense. Please check updates.